pkg/controller/installation/validation.go

```diff
@@ -108,14 +108,19 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 
 		nPools := len(instance.Spec.CalicoNetwork.IPPools)
 		if nPools > 2 {
-			return fmt.Errorf("Only one IPPool per version is allowed.")
+			return fmt.Errorf("Only one IPPool per version is allowed, pools: %+v.", instance.Spec.CalicoNetwork.IPPools)
```

Comment:
the changes in validation are to improve the info when hitting the error I saw ("Only one IPPool per version is allowed")